server/.env

```diff
@@ -1,2 +1,3 @@
-TEAM_NAME="Shums"
+TEAM_NAME="Punam"
 FLASK_ENV="development"
+MONGO_CONNECTION_STRING="mongodb+srv://Team-ravioli:ravioli1@tmravioli-tjfck.mongodb.net/test?retryWrites=true&w=majority"
```

Comment:
This seems to be a cloud setup, don't expose this on a public repo and share it with your teammates instead :)

Reply:
Thanks Bonnie for reminding me, i did it as you suggested.